src/status_im/navigation/core.cljs

```diff
@@ -92,6 +92,12 @@
  (fn [] (gesture/gesture-handler-root-hoc views/bottom-sheet))
  (fn [] views/bottom-sheet))
 
+;;;; Alert Banner
+(navigation/register-component
+ "alert-banner"
+ (fn [] (gesture/gesture-handler-root-hoc views/alert-banner))
```

Comment:
This most likely will intercept the touch outside the banner in iOS. We faced an issue with toast.
We need to add a workaround like we did for the toast:
```clojure
 #(gesture/gesture-handler-root-hoc views/toasts #js {:flex 0})
```

Reply:
done, thank you